extensions/data-workspace/yarn.lock

```diff
@@ -189,6 +189,27 @@
   dependencies:
     vscode-extension-telemetry "^0.1.6"
 
+"@nodelib/fs.scandir@2.1.4":
```

Comment:
What do these changes in this file mean?

Reply:
these are the dependences of the fast-glob package I added as a dependency for this extension. This file gets updated when you run yarn

Reply:
Sounds good. Thanks!